nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -26,6 +27,9 @@
 from nncf.common.tensor_statistics.statistic_point import StatisticPointsContainer
 from nncf.common.utils.backend import BackendType
 from nncf.common.utils.backend import get_backend
+from nncf.experimental.common.tensor_statistics import statistical_functions as s_fns
+from nncf.experimental.tensor import Tensor
+from nncf.experimental.tensor import functions as fns
```

Comment:
What is the difference between `fns` and `s_fns`? Why should we use all of them?

Reply:
It's suggestion of @alexsu52 https://github.com/openvinotoolkit/nncf/pull/2058#discussion_r1312954941

Idea to split functions that override functions under Tensor (like mean, min and etc) and functions that we use to calculate statistics that operating only wrapped Tensors.

Reply:
I still don't get when the `fns` and `s_fns` would be merged.

Reply:
What is reason to merge them?